go/tendermint/tendermint.go

```diff
@@ -645,11 +640,6 @@ func (t *tendermintService) getTendermintGenesis() (*tmtypes.GenesisDoc, error)
 		return nil, errors.Wrap(err, "tendermint: failed to create genesis doc")
 	}
 
-	// HACK: Certain test cases use TimeoutCommit < 1 sec, and care about the
```

Comment:
You did move these somewhere right?  (sorry github's review-view makes it hard to find stuff).

Reply:
Yeah, sorry, I moved this back now.